### PR TITLE
Doc: Clarify local vs online database usage

### LINKED ISSUE #4: Unclear Usage Instructions
Clarify usage instructions when using (local) `PosteriorDabase`.

The README has usage instructions that explain how to use the (local) `PosteriorDatabase` class. However, it does not clearly state that one must first _clone_ the original PosteriorDB GitHub repo from https://github.com/stan-dev/posteriordb. Thus naively installing `pip install posteriordb` and running the example with `PosteriorDatabase` fails.

This is especially confusing since the example states the default path is set to  `posteriordb/posterior_database`. However, this exact directory structure appears in both `posteriordb` and the interface `posteriordb-python`.

It would be especially helpful if the instructions clarified when discussing `posteriordb-python` and when discussing `posteriordb`.

Screenshot of the confusing instructions:
Screenshot 2024-09-25 at 3 32 48 PM

### PR DESCRIPTION
**Pull request by giladturok**
Clarify README instructions on using `posteriordb-python` with local vs online `posteriordb` access.

Closes #4 .

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,10 @@
 
 # `posteriordb-python`: a python library to work with `posteriordb`
 
-This repository contain the python library to easily work with the [posteriordb](https://github.com/stan-dev/posteriordb) repository. The library included database contain convenience functions to access data, model code and information for individual posteriors, models, data and reference draws.
+This repository contains the R package to efficiently work with the [posteriordb](https://github.com/stan-dev/posteriordb) repository. The R package includes convenience functions to access data, model code and information for individual posteriors, models, data and draws.
```
Comment (bob-carpenter) [inline README.md]: I'm confused---under the posteriordb-python heading it now says it contains an R package.  Am I missing something?

File: README.md [hunk 2]
```diff
@@ -27,22 +30,24 @@ pip install .
 
 ## Using the posterior database from python
 
-The included database contains convenience functions to access data, model code and information for individual posteriors.
+The included database contains convenience functions to access data, model code, and information for individual posteriors. We can create this database with either local or online access to `posteriordb`.
```
Comment (bob-carpenter) [inline README.md]: Ideally these things would not be written in the third person, not in second person singular (you/you/yours) or plural (we/us/ours).

**Review by bob-carpenter — CHANGES_REQUESTED** [review]
I don't think I should be the final person to review this, but I did have a couple of things.

File: README.md (hunk 1 above)
Comment (giladturok) [reply README.md]: Don't know how this snuck in there. Just fixed this.

File: README.md (hunk 2 above)
Comment (giladturok) [reply README.md]: Also fixed.

**Comment by MansMeg** [comment]
Excellent!

**Pull request merged**